openml_OS/models/Task.php

```diff
@@ -69,7 +69,7 @@ function search( $task_type_id, $keyValues ) {
     return $this->query( $sql );
   }
   
-  // @deprecated, source of errors
+  // @deprecated, source of errors. REMOVE ASAP
```

Comment:
Can we remove this now?

Reply:
I would be *highly* in favour of this, but the frontend task creation relies on this function.